src/v2/spec.yaml

```diff
@@ -5500,6 +5500,37 @@ paths:
           $ref: "#/components/responses/400Response"
         "404":
           $ref: "#/components/responses/404Response"
+  /farcaster/user/by_username:
+    get:
+      tags:
+        - User
+      summary: "Fetch a user by username"
+      description: "Fetches a single hydrated user object given a username"
+      operationId: user-by-username
```

Comment:
will be interesting to see how readme handles page route swapping and SEO for this one. hopefully no edge cases